src/getDescription.js

```diff
@@ -0,0 +1,31 @@
+export function getDescription() {
+  const metaDescription = document.querySelector(`meta[name="description"]`);
+  const metaOgDescription = document.querySelector(
+    `meta[property="og:description"]`
+  );
+
+  let metaDescriptionText = metaDescription
+    .getAttribute("content")
+    .replaceAll(/png|jpg|jpeg|gif/gi, " ");
+  let metaOgDescriptionText = metaOgDescription
+    .getAttribute("content")
+    .replaceAll(/png|jpg|jpeg|gif/gi, " ");
+
+  const metaFilterText = metaDescriptionText
+    .split(/[|!?|~,'"|/./]/)
+    .filter((text) => text !== "")
+    .map((blank) => blank.trim());
+  const metaOgFilterText = metaOgDescriptionText
+    .split(/[|!?|~,'"|/./]/)
+    .filter((text) => text !== "")
+    .map((blank) => blank.trim());
+
+  const finalFilterDescription = [];
+
+  metaDescription && finalFilterDescription.push(metaFilterText);
+  metaOgDescription && finalFilterDescription.push(metaOgFilterText);
+
+  if (metaDescription && metaOgDescription === false) {
```

Comment:
방금 찾아보니, 연산자 우선순위가 ===이 && 보다 더 높더라구요

그래서 테스트 해보니 metaDescription 과 metaOgDescription에 어떤 것을 넣더라도 모두 falsy가 나와요

그래서 혹시 둘 다 없는 경우라면 요건 어떨까요?
```suggestion
  if (!metaDescription && !metaOgDescription) {
```

Reply:
앗 그렇군요!! 제안 주신 코드로 보니 훨씬 깔끔하네요!! 하나 더 배우고 갑니다!